addons/OPT/REVIVE/fn_clientInitEH.sqf

```diff
@@ -121,6 +121,11 @@ DFUNC(playerHandleDamage) =
 		_unit setDamage GVAR(MAX_DAMAGE);		
 	};	
 
+	if (getDammage _unit >= GVAR(MAX_DAMAGE)) then 
```

Comment:
Was ist der Unterschied zwischen `_damage` und `getDammage unit` ?

Reply:
_damage ist der Schaden wert der vom EH übergeben wird. 
getDammage gibt den Schaden wieder den die Einheit hat wieder.

Reply:
Das war mir schon klar, das sehe ich ja oben auch in Zeile 112 und 116 :)

Ich wollte wissen in wiefern sich die beiden Variablen unterscheiden.

Wenn der Eventhandler nun auch einfach nur den Unit-Schaden zurückgibt ist es redundant.

Kann man sich die `playerHandleDamage_unit` bzw. `playerHandleDamage_damage` Objekte irgendwo anschauen?

Reply:
Der EH gibt den Schaden normaler weise zurück den die Einheit bei seiner Auslösung zugefügt bekommen hat.
mehr als das gibt es nicht https://community.bistudio.com/wiki/Arma_3:_Event_Handlers#HandleDamage